plugins/mac-videotoolbox/encoder.c

```diff
@@ -991,6 +1069,17 @@ static obs_properties_t *vt_properties(void *unused, void *data)
 	p = obs_properties_add_int(props, "keyint_sec", TEXT_KEYINT_SEC, 0, 20,
 				   1);
 	obs_property_int_set_suffix(p, " s");
+}
+
+static obs_properties_t *vt_properties_h264(void *unused, void *data)
+{
+	UNUSED_PARAMETER(unused);
+	struct vt_encoder_type_data *type_data = data;
+
+	obs_properties_t *props = obs_properties_create();
+	obs_property_t *p;
+
+	vt_properies_bitrate(props, data);
```

Comment:
@gxalpha Mark for refactoring later: Rename function to `vt_properties_bitrate`.